specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/cluster.json

```diff
@@ -659,6 +763,13 @@
         "clientGroupInfo": {
           "$ref": "#/definitions/ClientGroupInfo",
           "description": "The information of AAD security group."
+        },
+        "configurationOverride": {
+          "type": "object",
+          "additionalProperties": {
```

Comment:
Is configurationOverride a new property or an existing property? Always define strongly typed objects and avoid free-style objects.
Please refer here: https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi032-use-of-additionalproperties---always-define-strongly-typed-objects-and-avoid-free-style-objects

The comment is applicable to other places as well where new properties are added and additionalProperties is used.

Reply:
> Is configurationOverride a new property or an existing property? Always define strongly typed objects and avoid free-style objects.
> Please refer here: https://armwiki.azurewebsites.net/api_contracts/guidelines/openapi.html#oapi032-use-of-additionalproperties---always-define-strongly-typed-objects-and-avoid-free-style-objects
> 
> The comment is applicable to other places as well where new properties are added and additionalProperties is used.

Hi @RamyasreeChakka Thanks for your comment.
configurationOverride is an existing property in server side. Add this property is in order to fix S360 correctness issue.

Thanks for suggestion. I checked that this property is Dictionary<string, string> in backend, so the additionalProperties type is string.